scipy/stats/_continuous_distns.py

```diff
@@ -9008,10 +9009,15 @@ def _argcheck(self, beta, m):
 def _argus_phi(chi):
     """
     Utility function for the argus distribution
-    used in the CDF and norm of the Argus Funktion
+    used in the pdf, sf and moment calculation.
+    For small chi, use gammaincinv for better precision as for all x > 0:
+    gammainc(1.5, x**2/2) = 2 * (_norm_cdf(x) - x * _norm_pdf(x) - 0.5).
+    this can be verified directly by noting that the cdf of Gamma(1.5) can
+    be written as erf(sqrt(x)) - 2*sqrt(x)*exp(-x)/sqrt(Pi)
     """
-    return _norm_cdf(chi) - chi * _norm_pdf(chi) - 0.5
-
+    return _lazywhere(chi > 0.5, (chi,),
+                      lambda c: _norm_cdf(c) - c * _norm_pdf(c) - 0.5,
+                      f2=lambda c: sc.gammainc(1.5, c**2/2) / 2)
```

Comment:
Good point, the results in https://github.com/scipy/scipy/pull/13370#issuecomment-779443645 indicate that we might be able to use gammainc for all chi but I need to test it.

Reply:
gammainc is indeed faster, I tried a few more values like 1, 2.5, 5 and 10. So I changed the code and clean up the testing since the accuracy around 0.5 is no longer a concern (I left chi=0.5 in the test cases and removed chi=0.51)

Reply:
Huh. When I did a very limited test, i thought gammainc was slower for large chi and the same for small chi. Could you post the code you used to test?

Reply:
I used https://github.com/scipy/scipy/pull/13370#issuecomment-779443645, I tried with x=1, 2.5, 5 and 10 and every time, gamma was a bit faster, similar to the results in the comment

Reply:
The comment suggests that the test was performed with a single value of `x` or with 30 values in an array `x`. In this case the timing results may be dominated by the overhead of a function call. For larger arrays of `x` (e.g. 10000), `gammainc` is slower, but I don't think it's important enough to justify the use of `_lazywhere`.

Reply:
yes, I only tried it for small arrays / single values since that is the typical use case. unless you observed big difference when trying it on larger arrays, I would keep the implementation as it is. what do you think?

Reply:
`gammainc` took maybe 2 or 3 x longer for large arrays, but this is fine with me as-is.  If somebody wants to optimize this for speed, that can be a different project.